### ISSUE TITLE
Add support for `torch.nn.functional.normalize`

### ISSUE THREAD

**Issue by MiWeiss** [dotnet/TorchSharp#1382]
There is currently no support for `torch.nn.functional.normalize`. 

See also this comment: https://github.com/dotnet/TorchSharp/pull/707#issuecomment-2414260217

I'll have a go trying to open a PR with this functionality, but can't make any promises :-)
